packages/gatsby-theme-docs/src/layouts/internals/layout-header.js

```diff
@@ -116,17 +120,21 @@ const DocumentationSwitcherContainer = styled.div`
         ? designSystem.colors.light.linkNavigation
         : 'transparent'};
 
+  :hover,
+  :focus {
+    border-bottom: 2px solid ${designSystem.colors.light.linkNavigation};
+    color: ${designSystem.colors.light.linkNavigation};
+    svg {
+      * {
+        fill: ${designSystem.colors.light.linkNavigation};
+      }
+    }
+  }
+
   @media screen and (${designSystem.dimensions.viewports.desktop}) {
     margin: 0 0 0 ${designSystem.dimensions.spacings.xl};
   }
 `;
-const SwitcherButton = styled.a`
```

Comment:
This is not necessary anymore, I "merged" it into the `DocumentationSwitcherContainer`, which is now `DocumentationSwitcherButton`